application/recognition/image_recognition.py

```diff
@@ -0,0 +1,48 @@
+"""Image Recognition"""
+from eventlet import spawn_n
+from requests import post
+from application.helpers.artifact_helper import build_url
+from flask import current_app
+
+
+def recognize_image(file_url, object_id):
+    """Called by classes """
+    spawn_n(_work_asynchronously, file_url, object_id)
+
+
+def _call_google_api(url):
+    """Does API call to could vision api"""
+    api_url = current_app.config.get('CLOUD_VISION_API_URL')
+    api_key = current_app.config.get('CLOUD_VISION_API_KEY')
+    r = post(url=api_url,
```

Comment:
What if the image api is not available? Does anything significant happen? If it just crashes it's probably not too bad just want to prevent that image upload fails if the API is unavailable. But since it is in a second thread this probably doesn't brick that right?